README.md

```diff
@@ -64,6 +64,7 @@ This template comes with:
 - [Jbuilder](https://github.com/rails/jbuilder) for json views
 - [Letter Opener](https://github.com/ryanb/letter_opener) for previewing a mail in the browser
 - [Oj](https://github.com/ohler55/oj) for optimized json
+- [Pagy](https://github.com/ddnexus/pagy) for pagination
```

Comment:
and tell @jdrosales17 to update his post 😂

Reply:
Damn... now I definitely have to add this gem to the post 😆